packages/web-components/src/components/footer/legal-nav.ts

```diff
@@ -76,15 +79,18 @@ class C4DLegalNav extends StableSelectorMixin(LitElement) {
     return this.size !== FOOTER_SIZE.MICRO
       ? html`
           <nav
+            part="legal-nav"
             class="${c4dPrefix}--legal-nav"
             aria-label="${ifDefined(navLabel)}">
-            <div class="${this._getTypeClass()}">
+            <div part="legal-nav-list" class="${this._getTypeClass()}">
               <ul>
```

Comment:
Missing a `part` attribute here. I'd suggest naming the above `<div>` on line 85 as `legal-nav-list-container` and this`<ul>` take the name `legal-nav-list`.